www/include/common/common-Func.php

```diff
@@ -80,16 +80,6 @@ function returnSvg(string $svgPath, string $color, float $height, float $width):
     }
 }
 
-/*
- * Form Rules
- */
-function slash($elem = null)
```

Comment:
This function is used in this file `www/include/Administration/parameters/general/form.php`